src/components/table.js

```diff
@@ -260,6 +268,27 @@ export class Table extends Component {
       return 0;
   };
 
+  // sort data in same order as it was before, based on index
+  preserveSortData = (oldData, newData) => {
```

Comment:
So both `oldData` and `newData` are sorted arrays? What are their typical sizes? If their sizes are relatively small (say less than 100), then this brute-force implementation is okay; otherwise we may need to optimize the algorithm.

Reply:
They can be very large. Definitely larger than 100.

Reply:
I just stress tested it with 20,000 entries, and it took about 200ms. 20,000 entries should only occur here: https://het.io/disease-genes/browse/ for genes, and not in hetmech. And in the case of DAG, this function should never really be run because the table wont update with new data after its initialization. Therefore, I think this is okay.

Reply:
:+1: